.babelrc

```diff
@@ -3,6 +3,7 @@
     "@babel/preset-env"
   ],
   "plugins": [
-    "@babel/plugin-transform-runtime"
+    "@babel/plugin-transform-runtime",
+    "@babel/plugin-proposal-throw-expressions"
```

Comment:
Did we mean to push this up? I can recall adding this in hopes of fixing a local problem!